samples/src/main/java/com/mapbox/samples/BasicV6BatchGeocoding.java

```diff
@@ -36,11 +36,11 @@ private static List<V6RequestOptions> requestOptions() {
       .street("Pennsylvania Avenue NW")
       .postcode("20500")
       .place("Washington, DC")
+      .country("us")
       .build();
 
     final V6ForwardGeocodingRequestOptions structuredInputOptions = V6ForwardGeocodingRequestOptions
       .builder(structuredInputQuery)
-      .country("us")
```

Comment:
Country in `V6ForwardGeocodingRequestOptions` is still supported, right? So you can leave country in this example